### PR TITLE
complete CSV dataframe interface

### PR DESCRIPTION
**Pull request by PelerYuan**

### CODE REVIEW

File: utils/dataFrame.go [hunk 1]
```diff
@@ -47,27 +49,53 @@ func (c *CSVDataBase) CloseDB() error {
...
+	return c.csvData[RawIndex], nil
 }
 
-func (c *CSVDataBase) GetCellData(key string, keyColNum int, valueColNum int) []string {
-	return nil
+func (c *CSVDataBase) GetCellData(key string, RawIndex int) (string, error) {
+	found := false
+	index := 0
+	for index = range c.csvData[0] {
+		if c.csvData[index][index] == key {
```
Comment (Tommy-Zhang3759) [inline utils/dataFrame.go]: do you mean csvData[Rawindex][index]? by the way, row, instead of "raw" maybe.

File: utils/dataFrame.go [hunk 2]
```diff
@@ -47,27 +49,53 @@ func (c *CSVDataBase) CloseDB() error {
...
+	c.csvData[RawIndex] = data
 	return nil
 }
 
-func (c *CSVDataBase) SaveCellData(key string, keyColNum int, valueColNum int) error {
+func (c *CSVDataBase) SetCellData(key string, RawIndex int, data string) error {
+	found := false
+	index := 0
+	for index = range c.csvData[0] {
+		if c.csvData[index][index] == key {
```
Comment (Tommy-Zhang3759) [inline utils/dataFrame.go]: might same as the previous problem. [RawIndex][index] instead of [index][index]

**Review by Tommy-Zhang3759 — CHANGES_REQUESTED** [review]
calm down~ if you don't want, no need to spend time. just~ be~ happy~~~

**Comment by PelerYuan** [comment]
apologize for disatraction...
I will rewrite all the code

**Pull request merged**